ert/ensemble_evaluator/builder/_job.py

```diff
@@ -107,14 +118,16 @@ def set_args(self, args: Tuple[str, ...]) -> "_JobBuilder":
     def build(self) -> Union[_FunctionJob, _UnixJob]:
         if self._id is None:
             self._id = str(uuid.uuid4())
+        if self._index is None:
+            self._index = "0"
```

Comment:
I think we should require this? Fail if not set?

Reply:
It was failing when haven't specified it though; ie. failing anyway. I have a question: why `self._id` can be None also?

Reply:
I think it's just tailored for Prefect, so that Prefect can get IDs generated… No other reasons IIRC.

Reply:
Now it's mandatory